Source/ARTPush.m

```diff
@@ -179,7 +179,7 @@ + (void)didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceTokenDa
     NSString *deviceToken = [hexString copy];
 
     [rest.logger info:@"ARTPush: device token: %@", deviceToken];
-    NSString *currentDeviceToken = [rest.storage objectForKey:ARTDeviceTokenKey];
+    NSString *currentDeviceToken = [rest.storage objectForKey:ARTAPNSDeviceTokenKey];
```

Comment:
Rename it to`currentAPNSDeviceToken`

Reply:
This is in the context of `didRegisterForRemoteNotificationsWithDeviceToken` which is already APNS, so I think it's okay here.